jbehave-support-core/src/test/groovy/org/jbehavesupport/test/sample/SshReport.story

```diff
@@ -32,3 +32,11 @@ Then the following data are present in [TEST] log:
 | header     |
 | long       |
 | unexpected |
+
+Scenario: Save more values into context to demonstrate Test Context extension behaviou
```

Comment:
Minor: do we want it in ssh report story or somewhere else?

Reply:
I was thinking about renaming SshReport story to Report stoy. We doesn't have any another sotry representing report.